src/components/Play/Comments/Comments.js

```diff
@@ -0,0 +1,50 @@
+"use client";
+
+import contentsData from "src/mock/contents.json";
+import { useState } from "react";
+import "./Comments.css";
+
+export default function Comments(props) {
+  // var commentsId = props.commentsId;
+  var content = contentsData[props.commentsId];
+  var comments = content.comments;
+
+  var [newComment, setNewComment] = useState("");
+
+  if (comments == 0) {
+    return (
+      <div className="comments-container">
+        <p className="comment-number">댓글 0개</p>
+        <input
+          className="comment-input"
+          placeholder="댓글을 작성하세요"
+          value={newComment}
+          onChange={(e) => {
+            setComment(e.target.value);
```

Comment:
```suggestion
            setNewComment(e.target.value);
```

함수명때문에 여기 오류가 뜨네요! 고쳐야 할 듯